lib/js/dom/structure.js

```diff
@@ -0,0 +1,118 @@
+// The controls above the input panel
+const btnReset = {
+  tagName: 'button',
+  className: 'btn-alt btn-reset js-btn-reset',
+  textContent: 'Reset'
+};
+
+const btnShortUrl = {
+  tagName: 'button',
+  className: 'btn-alt btn-short-url js-btn-short-url',
+  textContent: 'Make Short URL:'
+};
+
+const urlOut = {
+  tagName: 'input',
+  className: 'url-out js-url-out',
+  attrs: {
+    type: 'text'
+  }
+};
+
+// The input panel
+const inputPanel = {
```

Comment:
these configs might be nice `rdom`-style, see https://www.npmjs.com/package/rdom
you could wrap it in a thunk to defer evaluation